test/unit/rules/no-forbidden-elements-test.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+const ERROR_MESSAGE_FORBIDDEN_ELEMENTS = require('../../../lib/rules/no-forbidden-elements')
+  .ERROR_MESSAGE_FORBIDDEN_ELEMENTS;
+
+generateRuleTests({
+  name: 'no-forbidden-elements',
+
+  config: true,
+
+  good: ['<header></header>', '<div></div>', '<footer></footer>', '<p></p>'],
+
+  bad: [
```

Comment:
Can you add `bad` examples where a user configures a custom element? Something like:

```
{
  config: ['Foo'],
  template: '<Foo />',
  result: {
    // error....
  }
}
```

Reply:
Added